internal/map_range_to_range_test.go

```diff
@@ -0,0 +1,11 @@
+package internal
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestMapRangeToRange(t *testing.T) {
```

Comment:
Please add some more tests with negative values, and float values (123.5, 123.324, etc.)